FlexMeasures/flexmeasures
# Add histogram option on the sensor page

## Description

I have add a histogram option on the sensor page, and also refactored the code. 

## Look & Feel

![image](https://github.com/user-attachments/assets/a5efbf58-efe7-4094-8413-54889d7e44fc)

## How to test

- Check out this branch and load any sensor. Then select `Histogram` from the `Select Chart` dropdown and see the chart. 

---
